screens/EventDetails/LocationSection.tsx

```diff
@@ -55,7 +53,7 @@ const LocationSection = ({
         `${coordinates.latitude}, ${coordinates.longitude}`
       )
     }
-    showToast("Copied to Clipboard", bottomTabHeight)
```

Comment:
Nice improvements here, removing the extra bottomTabHeight param makes it a lot more readable/easier to edit later